README.md

```diff
@@ -577,6 +579,16 @@ Installs [stern](https://github.com/stern/stern), a multi pod and container log
 * DEBUG_stern: Debug this feature
 * Environment STERN_VERSION (required): Valid Stern version (e.g. 1.21.0)
 
+### <a id="tanzucli"></a> Tanzu CLI
+
+Installs the [Tanzu CLI](https://github.com/vmware-tanzu/tanzu-cli)
+
+#### Configuration
+
+* USE_tanzucli: Enable this feature
```

Comment:
Why still use these options?

Reply:
Both FEATURES and USE_ are supported and the documentation generator creates this line.

Reply:
Yes, that is obvious. The question is: why?

Reply:
Backwards compability? The userbase of cloudcontrol might not be huge, but there is one. I think this backwards compability does not hurt, especially when there is a generator behind it.
That is my opinion.

Reply:
Then deprecate it and remove them at the next opportunity.

Reply:
We can't remove it as the test suite uses the feature for example. What we could do would be to change the documentation to either remove the USE_ line or add a note about the FEATURES env variable.

But I would rather put that into another PR.

Reply:
ok. In my opinion the new interface is prettier and I wanted to avoid superfluous code. BUT: that's just my opinion, so do as you like.